tests/interop/go-tuf-transition-M4/generate.go

```diff
@@ -0,0 +1,146 @@
+package main
```

Comment:
Instead of having this file, I think it'd make more sense to leave a README here explaining where this metadata came from

Reply:
Likewise, I removed the go file, added README, and renamed this `fuchsia-go-tuf-transition-M4`.